js/plugins/trimTimeline.js

```diff
@@ -12,6 +12,28 @@ var trimClinicalTimeline = function (maxDays, minDays, getZoomLevel, width, getT
   var zoomLevel = getZoomLevel(minDays, maxDays, width);
   var tickValues = getTickValues(minDays, maxDays, zoomLevel);
 
+  var getDiffernceTicksDays = function (zoomLevel) {
```

Comment:
This function feels like it should be shared between plugins and the timeline. Maybe some separate `util.js` or `util/ticks.js` file. Also: Differnce -> Difference